lib/LaTeXML/Engine/LaTeX.pool.ltxml

```diff
@@ -2804,8 +2809,9 @@ DefMacro('\UseTextAccent{}{}', '{\fontencoding{#1}#2{#3}}');
 DefPrimitive('\DeclareMathAccent DefToken {}{} {Number}', sub {
     my ($stomach, $cs, $kind, $class, $code) = @_;
     $class = ToString($class);
-    my $info  = LookupValue('fontdeclaration@' . $class);
-    my $glyph = FontDecode($code->valueOf, ($info ? $$info{encoding} : $class));
+    my $info = LookupValue('fontdeclaration@' . $class);
+    #    my $glyph = FontDecode($code->valueOf, ($info ? $$info{encoding} : $class));
+    my ($glyph) = FontDecode($code->valueOf, ($info ? $$info{encoding} : $class));
```

Comment:
This array unwrap looks a little scary/fragile...
Should we check for `wantsarray` in FontDecode? 

We could even avoid computing the font details if we are only requesting the scalar `$glyph` back.

Reply:
I'd been tempted to minimize the change to the API by doing that, but Perl tricks you. A previous call had been `Box(FontDecode(...),...)` where you'd think you were getting *only* the `$glyph`, but actually wantarray is true!

Reply:
Well the `Box()` auto-unlisting pattern is also a dangerous thing to do in Perl :>

`Box(scalar(FontDecode(...)),...` doesn't have quite the same ring to it, I admit.